stock_picking_tracking/models/stock_picking_tracking_group.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+from .stock_picking_tracking_event import OPERATION_STATES
```

Comment:
As you want to share the same states across several objects, this is not the best strategy, as on inheritance you will lose the same states on all places. Better to use this technique: https://github.com/OCA/sale-workflow/blob/eb94d71ba9f75a0f84c9435b6dac10c010a35aa7/sale_order_type/models/sale_order_type.py#L20 (it also handles translations)